### DISCUSSION TITLE
How to set component like Q-Table to 90% width

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [quasarframework/quasar#12339] [General - Components / Directives / etc]
I cannot find how to set the width of a component like Q-Table to 90% of the current page.

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/WNXGGba?editors=1010](https://codepen.io/milospaunovic/pen/WNXGGba?editors=1010)

So, either  set a style attribute for `QTable` or create a new class for it, or tinker with Quasar classes.

**Reply by michaeloliveraz** [reply]
Thanks that's it.
